### PR TITLE
Add description to mother language day post.

### PR DESCRIPTION
**Pull request by robdodson**
I noticed the mother language day post's twitter card was showing up with the placeholder description in the twitter card validator.

image

### CODE REVIEW

File: site/en/blog/mother-language-day-2021/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: 'layouts/blog-post.njk'
 title: International Mother Language Day 2021
 description: >
-  Describe your example post with a couple sentences.
+  International Mother Language Day is an annual celebration of cultural diversity observed worldwide on February 21.  It promotes the preservation and protection of languages spoken all over the world and celebrates multilingualism.
```
Comment (robdodson) [inline site/en/blog/mother-language-day-2021/index.md:5]:
```suggestion
  International Mother Language Day is an annual celebration of cultural diversity observed worldwide on February 21. It promotes the preservation and protection of languages spoken all over the world and celebrates multilingualism.
```

**Comment by robdodson** [comment]
@surma just a heads up that I'm adding this. Let me know if you'd prefer a different description.

**Review by samthor — APPROVED** [review]
Seems worth getting in even without author review.

**Comment by surma** [comment]
Oh god thank you. That’s definitely an oversight. I had a summary before, but I started over because of build system issues and must have forgotten about this. Thanks y’all!

**Pull request merged**